WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsView.swift

```diff
@@ -0,0 +1,213 @@
+import SwiftUI
+
+struct PointOfSaleSettingsView: View {
+    @State private var selection: SidebarNavigation? = .store
+
+    var body: some View {
+        HStack(spacing: POSSpacing.none) {
+            VStack {
+                List(selection: $selection) {
+                    Section {
+                        ForEach([SidebarNavigation.store, SidebarNavigation.hardware], id: \.self) { item in
+                            HStack {
+                                Image(systemName: item.icon)
+                                    .font(.posBodyLargeRegular())
+                                VStack(alignment: .leading) {
+                                    Text(item.title)
+                                        .font(.posBodyLargeRegular())
+                                    Text(item.subtitle)
+                                        .font(.posBodyMediumRegular())
+                                        .foregroundStyle(.secondary)
+                                }
+                            }
+                            .tag(item)
+                        }
+                    }
+                }
+                .safeAreaInset(edge: .bottom) {
```

Comment:
I wasn't able to pin the `Help` section to the bottom of the view while this is part of a `List`. This approach worked, but I'm not fully convinced is the best way to move forward.

We do not necessarily need a list here and perhaps creating just a reusable "card" would work fine for the 3 items. If we do, this can be iterated on a separate PR. Thoughts?

Reply:
Agreed, if a list makes it harder, let's just make our own cards.

Also, this _might_ move the help button above an on-screen keyboard, which would be a bit strange. It's likely that we'll end up presenting one in settings eventually.